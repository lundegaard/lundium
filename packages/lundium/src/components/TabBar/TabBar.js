import React from 'react';
import PropTypes from 'prop-types';
import { cx } from 'ramda-extension';

import Box from '../Box';

const TabBar = ({ className, ...rest }) => (
	<Box className={cx('tab-bar', className)} {...rest} />
);

TabBar.propTypes = {
	className: PropTypes.string,
	stepWizzard: PropTypes.bool,
};

export default TabBar;
