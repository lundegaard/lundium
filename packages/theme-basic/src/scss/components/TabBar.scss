@import '../core';

.tab-bar {
	position: relative;
	display: flex;
	overflow-x: auto;
}