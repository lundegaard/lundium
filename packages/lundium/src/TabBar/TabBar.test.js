import React from 'react';
import { shallow } from 'enzyme';

import TabBar from './TabBar';

describe('<TabBar />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<TabBar {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
