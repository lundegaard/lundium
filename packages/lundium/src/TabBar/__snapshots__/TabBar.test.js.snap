// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabBar /> matches snapshot when shallowly rendered 1`] = `
<forwardRef(Box)
  className="tab-bar"
/>
`;
